templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>

```

Comment:
The links in the sidebar are currently placeholders (`#`). Consider replacing them with actual URLs or Django template tags that link to the appropriate views in your application.